packages/app/src/App.tsx

```diff
@@ -107,7 +112,15 @@ const routes = (
     </Route>
     <Route path="/catalog-import" element={<CatalogImportPage />} />
     <Route path="/docs" element={<TechdocsPage />} />
-    <Route path="/create" element={<ScaffolderPage />} />
+    <Route
```

Comment:
Thinking this would be nicer instead, thoughts?

```tsx
<Route path="/create" element={<ScaffolderPage />}>
  <OwnerPicker />
  <RepoUrlPicker />
</Route>
```

Potentially wrapping the form field extensions in another element for clarity, as discussed in the issue.

Reply:
These extensions - the names become a bit confusing to me. Having a `<OwnerPicker />` child tells me that it's somehow rendered "directly". If it were at least named `OwnerPicker.Extension` or `OwnerPickerExtension` or `OwnerPickerScaffolderAddon` it would feel like it made a ton more sense, just when reading the code.

I think this calls for some form of naming standard

Reply:
I think it might make a little more sense when it's wrapped in it's `slot`(ish) wrapper.

```ts
<Route path="/create" element={<ScaffolderPage />}>
  <Scaffolder.CustomFields>
    <OwnerPicker />
    <RepoUrlPicker />
  </Scaffolder.CustomFields>
</Route>
```

Reply:
Or do you still think there's room for improvement @freben?

Reply:
Still wondering if a suffix on their names makes sense, to explain what they do. Just looking at the line `import { OwnerPicker }` at the top gives no indication of how it's _used_, if that makes sense? It's intended to be used as an addon for something, but has no indication that that's the case.

I'm just pondering if it would make sense to make it a standard to name all extension things like this, everywhere, for all purposes, `*Extension` or something :) Not sure